src/ShortcutsManager.cpp

```diff
@@ -30,6 +30,7 @@ void ShortcutsManager::setShortcut(const QString& key, QKeySequence* sequence)
 {
     QKeySequence* newSequence = new QKeySequence(*sequence);
     shortcuts.value(key)->swap(*newSequence);
+    delete newSequence;
```

Comment:
Using `std::unique_pointer` is another, more modern way to address this as well - it'll automatically take create of deleting stuff when it goes out of scope

Reply:
Informational, if this works it works